AppCenterReactNativeShared/android/build.gradle

```diff
@@ -111,7 +111,7 @@ allprojects {
 }
 
 dependencies {
-    api 'com.microsoft.appcenter:appcenter:2.3.0'
+    api 'com.microsoft.appcenter:appcenter:2.4.0'
```

Comment:
Is this correct?

Reply:
It was updated by bumping android sdk version, and this android sdk release is 2.4.0.